src/status_im2/common/bottom_sheet/sheets.cljs

```diff
@@ -47,6 +51,12 @@
 
           (= view :pinned-messages-list)
           (merge {:content pin.list/pinned-messages-list}))]
-    [bottom-sheet/bottom-sheet opts
-     (when content
-       [content (when options options)])]))
+    [:f>
+     (fn []
+       (rn/use-effect (fn []
+                        (rn/hw-back-add-listener dismiss-bottom-sheet-callback)
```

Comment:
`BackHandler` is only meaningful for Android. How does this `use-effect` is working for iOS? Shouldn't we check if the platform is Android before using `BackHandler`? Reference: https://reactnative.dev/docs/backhandler#addeventlistener